.github/workflows/go-action.yml

```diff
@@ -16,4 +15,4 @@ jobs:
         uses: dsp-testing/go-snapshot-action@lsep/go-list-impl
```

Comment:
I think this needs to be updated.

Reply:
_Good catch_. I knew it was too good to be true that it worked on the first run 🤦

Reply:
Oh! But it did work! :tada: https://github.com/dsp-testing/go-snapshot-action/runs/6836365970?check_suite_focus=true

Updating to point to `main`, which it will run once merged